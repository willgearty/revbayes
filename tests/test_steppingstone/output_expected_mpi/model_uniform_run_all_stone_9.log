Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-33879.77	-33878.21	-1.56114	1.420467	0.0677324	0.131068	0.0414309	0.0277295	0.0355071	0.231511	0.330589	0.06587	0.0386844	0.110811	0.114465	0.137301	0.0721032	0.147672	0.0885156	0.684599	0.197562	0.0185895	0.0786182	0.297012	0.101997	0.356976	0.153571	0.06263	0.0561222	0.0157717	0.134888	0.17763	0.00167044	0.101087	0.0341848	0.0656474	0.382103	0.00536458	0.0246454	0.0914906	0.0111052	0.142059	0.00354685	0.0272465	0.058953	0.100481	0.185652	0.381009	0.20814	0.13628	0.0490963	0.0654045	0.160069	0.212771	0.57476	1.04539	2.16708	0.0697801	0.738477	0.135812	0.0559314	0.5960343	5.212194
10	-29415.34	-29419.22	3.877187	1.010783	0.160522	0.0513803	0.15383	0.105917	0.0498188	0.0686928	0.237907	0.0314258	0.0665707	0.0344512	0.0908161	0.124931	0.0244643	0.0553731	0.0547954	0.235616	0.148747	0.0172078	0.323128	0.111282	0.0644661	0.262995	0.043942	0.018033	0.14163	0.0080282	0.370119	0.100342	0.00336324	0.344036	0.0110124	0.234206	0.192177	0.00643887	0.0465301	0.182441	0.0120337	0.0348252	0.00285949	0.0230476	0.0473166	0.113272	0.241877	0.34556	0.236359	0.186148	0.0631286	0.0806183	0.088186	0.1391	0.479969	1.00176	2.37917	0.0886439	0.628695	0.108015	0.174646	0.6180258	4.651868
