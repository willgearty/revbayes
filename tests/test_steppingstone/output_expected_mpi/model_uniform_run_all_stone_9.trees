Iteration	Posterior	Likelihood	Prior	psi
0	-33879.77	-33878.21	-1.56114	(Galeopterus_variegatus[&index=23]:0.153571,((Chlorocebus_aethiops[&index=5]:0.035507,((Nycticebus_coucang[&index=15]:0.088516,(Tarsius_syrichta[&index=21]:0.101997,Varecia_variegata_variegata[&index=22]:0.356976)[&index=28]:0.177630)[&index=36]:0.091491,Propithecus_coquereli[&index=19]:0.078618)[&index=29]:0.001670)[&index=37]:0.011105,(Colobus_guereza[&index=6]:0.231511,Aotus_trivirgatus[&index=1]:0.067732)[&index=42]:0.100481)[&index=25]:0.056122,(Otolemur_crassicaudatus[&index=16]:0.684599,(((((Galago_senegalensis[&index=8]:0.065870,(Daubentonia_madagascariensis[&index=7]:0.330589,(Microcebus_murinus[&index=14]:0.147672,Cheirogaleus_major[&index=4]:0.027729)[&index=39]:0.003547)[&index=32]:0.065647)[&index=40]:0.027247,Lepilemur_hubbardorum[&index=11]:0.114465)[&index=41]:0.058953,(Pan_paniscus[&index=17]:0.197562,Perodicticus_potto[&index=18]:0.018590)[&index=30]:0.101087)[&index=26]:0.015772,(Callicebus_donacophilus[&index=2]:0.131068,((((Loris_tardigradus[&index=12]:0.137301,Hylobates_lar[&index=9]:0.038684)[&index=31]:0.034185,Cebus_albifrons[&index=3]:0.041431)[&index=27]:0.134888,Saimiri_sciureus[&index=20]:0.297012)[&index=35]:0.024645,Macaca_mulatta[&index=13]:0.072103)[&index=33]:0.382103)[&index=34]:0.005365)[&index=24]:0.062630,Lemur_catta[&index=10]:0.110811)[&index=43]:0.185652)[&index=38]:0.142059)[&index=44];
10	-29415.34	-29419.22	3.877187	(Lemur_catta[&index=10]:0.034451,Galeopterus_variegatus[&index=23]:0.043942,(Otolemur_crassicaudatus[&index=16]:0.235616,(Cheirogaleus_major[&index=4]:0.105917,((Microcebus_murinus[&index=14]:0.055373,(Varecia_variegata_variegata[&index=22]:0.262995,((Tarsius_syrichta[&index=21]:0.064466,Nycticebus_coucang[&index=15]:0.054795)[&index=29]:0.003363,((Perodicticus_potto[&index=18]:0.017208,(Cebus_albifrons[&index=3]:0.153830,Galago_senegalensis[&index=8]:0.031426)[&index=41]:0.047317)[&index=32]:0.234206,((Callicebus_donacophilus[&index=2]:0.051380,(Saimiri_sciureus[&index=20]:0.111282,Loris_tardigradus[&index=12]:0.124931)[&index=27]:0.370119)[&index=33]:0.192177,(Lepilemur_hubbardorum[&index=11]:0.090816,(Hylobates_lar[&index=9]:0.066571,(Daubentonia_madagascariensis[&index=7]:0.237907,Chlorocebus_aethiops[&index=5]:0.049819)[&index=24]:0.018033)[&index=35]:0.046530)[&index=31]:0.011012)[&index=36]:0.182441)[&index=28]:0.100342)[&index=25]:0.141630)[&index=40]:0.023048)[&index=39]:0.002859,((Macaca_mulatta[&index=13]:0.024464,(Aotus_trivirgatus[&index=1]:0.160522,(Colobus_guereza[&index=6]:0.068693,Propithecus_coquereli[&index=19]:0.323128)[&index=42]:0.113272)[&index=37]:0.012034)[&index=26]:0.008028,Pan_paniscus[&index=17]:0.148747)[&index=30]:0.344036)[&index=34]:0.006439)[&index=38]:0.034825)[&index=43]:0.241877)[&index=44];
