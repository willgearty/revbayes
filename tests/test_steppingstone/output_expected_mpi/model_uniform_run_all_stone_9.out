state	power	likelihood
4	1e-05	-50045
5	1e-05	-30559.7
6	1e-05	-30161.8
7	1e-05	-28925.4
8	1e-05	-29501.4
9	1e-05	-29324.6
10	1e-05	-29419.2
